test/jdk/java/text/testlib/IntlTest.java

```diff
@@ -140,14 +105,46 @@ protected void run(String[] args) throws Exception
                 System.out.println("Exception: " + e.toString() + e.getMessage());
             }
         }
-        if (nothrow) {
-            if (exitCode) {
-                System.exit(errorCount);
-            }
-            if (errorCount > 0) {
-                throw new IllegalArgumentException("encountered " + errorCount + " errors");
+        if (exitCode) {
```

Comment:
@naotoj Since I got rid of `writeTestResult(errorCount);`

I will just have this block execute regardless of `nothrow`. All tests should now write the final result.